ci/jenkins/pipelines/prs/skuba-test.Jenkinsfile

```diff
@@ -89,10 +90,14 @@ node('caasp-team-private-integration') {
            }
            if (pr_repo_label != null) {
                def branch_name = pr_repo_label.name.split(":")[1]
-               branch_repo = "http://download.suse.de/ibs/Devel:/CaaSP:/${repo_version}:/Branches:/${branch_name}/SLE_15_SP2"
-               branch_registry = "registry.suse.de/devel/caasp/4.5/branches/${branch_name}/containers"
-               original_registry = "registry.suse.de/devel/caasp/4.5/containers/containers"
-           }
+               if (env.CHANGE_TARGET.startsWith('maintenance')){
```

Comment:
Should this also check if it includes "4.2" in the string?  At some point, I assume 4.5 will also be maintenance, as we move to integrating with Rancher.

I'm not sure what happens to skuba at that point, and no one else is either. :)  But this is sort of a hidden thing I'm not sure any of us will remember to update if that happens, so I don't know if it makes more sense to prepare for that possibility by making the code slightly more complex, or if there should be a document somewhere recording "things we will have to remember to change if 4.5 has a maintenance branch", or something else.  Or maybe this concern is all just premature optimization. :D

Reply:
It's a good point, but this is an unlikely scenario. We can manage it if this ever happens